### PR TITLE
fix: Cypress test for widget sidebar in Airgap

### PR DESCRIPTION
**Pull request by dhruvikn**
## Description
- Fixes an issue where widgets from WDS were showing up in Airgap.
- Remove the map widget from the widgets list in the widget sidebar cypress test.

#### PR fixes following issue(s)
Fixes # (issue number)
> if no issue exists, please create an issue and ask the maintainers about this first
>
>
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update
>
>
>
- [ ] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by Aishwarya-U-R — COMMENTED** [review]
Approving Cypress changes.

File: app/client/cypress/e2e/Regression/ClientSide/Debugger/Widget_property_navigation_spec.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as _ from "../../../../support/Objects/ObjectsCore";
 import OneClickBindingLocator from "../../../../locators/OneClickBindingLocator";
 
-describe("Widget property navigation", () => {
+describe("excludeForAirgap", "Widget property navigation", () => {
   it("Collapsed field navigation", () => {
```
Comment (berzerkeer) [inline app/client/cypress/e2e/Regression/ClientSide/Debugger/Widget_property_navigation_spec.ts:5]: This entire test should be skipped for airgap ?

**Review by berzerkeer — APPROVED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Debugger/Widget_property_navigation_spec.ts (hunk 1 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/Debugger/Widget_property_navigation_spec.ts:5]: Yes, we have confirmation from @akash-codemonk

Comment (berzerkeer) [reply app/client/cypress/e2e/Regression/ClientSide/Debugger/Widget_property_navigation_spec.ts:5]: Thanks !

**Review by jsartisan — APPROVED** [review]

**Comment by jsartisan** [comment]
Waiting for one final check on the DP. Let's wait for DP.

**Comment by jsartisan** [comment]
LGTM!

**Pull request merged**